src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -132,11 +140,36 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         return new ResponseEntity<>(studyService.delete(UUID.fromString(id), getAuthenticatedInfo()), HttpStatus.OK);
     }
 
-    public ResponseEntity<List<StudySummaryModel>> enumerateStudies(
-            @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
-            @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit) {
-        validiateOffsetAndLimit(offset, limit);
-        return new ResponseEntity<>(studyService.enumerate(offset, limit), HttpStatus.OK);
+    private void validateEnumerateParams(Integer offset, Integer limit, String sort, String direction) {
+        List<String> errors = new ArrayList<>();
+        if (offset < 0) {
+            errors.add("offset must be greater than or equal to 0.");
+        }
+        if (limit < 1) {
+            errors.add("limit must be greater than or equal to 1.");
+        }
+        if (sort != null && sort.length() > 0 && !VALID_SORT_OPTIONS.contains(sort)) {
+            errors.add(String.format("sort must be one of: (%s).", String.join(", ", VALID_SORT_OPTIONS)));
+        }
+        if (direction != null && direction.length() > 0 && !VALID_DIRECTION_OPTIONS.contains(direction)) {
+            errors.add("direction must be one of: (asc, desc).");
+        }
+        if (!errors.isEmpty()) {
+            throw new ValidationException("Invalid enumerate parameter(s).", errors);
+        }
+    }
+
+    public ResponseEntity<EnumerateStudyModel> enumerateStudies(
+            @Valid @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
+            @Valid @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit,
+            @Valid @RequestParam(value = "sort", required = false, defaultValue = "created_date") String sort,
+            @Valid @RequestParam(value = "direction", required = false, defaultValue = "asc") String direction,
```

Comment:
this shows the oldest first. should the default be the newest first?

Reply:
I had it the other way but then I thought about what it would be like for consumers of the API calling this with a sort param and no direction -- it would be weird to have things coming back default descending. We will explicitly request sort=created_date and direction=desc from the UI.